src/Paket/Program.fs

```diff
@@ -918,3 +918,5 @@ let main() =
         else printError exn
 
 main()
+
+ignore ()
```

Comment:
?

Reply:
@forki That's one of the things to.... ignore. ;-) I used that to set a final breakpoint for debugging and didn't want to sort out what to commit while this is still unfinished. As I said, I'll clean everything up when I finish the PR.